modules/effects/migrations/9_0_0/index.ts

```diff
@@ -0,0 +1,112 @@
+import {
+  chain,
+  Rule,
+  SchematicContext,
+  Tree,
+} from '@angular-devkit/schematics';
+import {
+  commitChanges,
+  visitTSSourceFiles,
+} from '@ngrx/effects/schematics-core';
+import {
+  createReplaceChange,
+  ReplaceChange,
+} from '@ngrx/effects/schematics-core';
+import * as ts from 'typescript';
+
+function renameErrorHandlerConfig(): Rule {
+  return (tree: Tree, ctx: SchematicContext) => {
+    visitTSSourceFiles(tree, sourceFile => {
+      const changes: ReplaceChange[] = replaceEffectConfigKeys(
+        sourceFile,
+        'resubscribeOnError',
+        'useEffectsErrorHandler'
+      );
+
+      commitChanges(tree, sourceFile.fileName, changes);
+
+      if (changes.length) {
+        ctx.logger.info(
+          `[@ngrx/effects] Updated Effects configuration, see the migration guide (https://ngrx.io/guide/migration/v9#effects) for more info`
+        );
+      }
+    });
+  };
+}
+
+function replaceEffectConfigKeys(
+  sourceFile: ts.SourceFile,
+  oldText: string,
+  newText: string
+): ReplaceChange[] {
+  const changes: ReplaceChange[] = [];
+
+  ts.forEachChild(sourceFile, node => {
+    visitCreateEffectFunctionCreator(node, createEffectNode => {
+      // only search the second arg to createEffect() (the config object)
+      if (createEffectNode.arguments[1]) {
```

Comment:
Personally I would rather see this as:

```ts
const [configNode] = createEffectNode.arguments
```

This also makes the comment obsolete

Reply:
yep done